app/components/events/view/overview/general-info.js

```diff
@@ -1,5 +1,12 @@
 import Component from '@ember/component';
 
 export default Component.extend({
-  classNames: ['ui', 'fluid', 'card']
+  classNames : ['ui', 'fluid', 'card'],
+  actions    : {
+    openModal() {
+      if (this.get('session.isAuthenticated')) {
```

Comment:
why this check?

Reply:
I added this check so that only authenticated user can check it. Just realised only authenticated users can access that page. I'll fix it.